### PR TITLE
NEW: Added decoding of repo_dir for get_git_repo_dir.

### PR DESCRIPTION
**Pull request by amakhnach**

### CODE REVIEW

File: pkgversion/pkgversion.py [hunk 1]
```diff
@@ -27,7 +28,10 @@ def get_git_repo_dir():
     repo_dir = repo_dir.strip()
     if not repo_dir:
         repo_dir = None
-    return repo_dir
+
+    country_code, encoding = locale.getdefaultlocale()
+
+    return repo_dir.decode(encoding) if repo_dir and encoding else repo_dir
```
Comment (sergray) [inline pkgversion/pkgversion.py]: That returns bytestring in Python 3, if encoding is unavailable

Comment (amakhnach) [inline pkgversion/pkgversion.py]: Yes, I can fix `.travis.yml` or we need different solution to get `str` from `bytestring`

File: pkgversion/pkgversion.py [hunk 2]
```diff
@@ -27,7 +28,11 @@ def get_git_repo_dir():
     repo_dir = repo_dir.strip()
     if not repo_dir:
         repo_dir = None
-    return repo_dir
+
+    country_code, encoding = locale.getdefaultlocale()
```
Comment (sergray) [inline pkgversion/pkgversion.py]: let's simplify it to

```
encoding = locale.getpreferredencoding()
```

it should allow to remove the line bellow as well

File: pkgversion/pkgversion.py [hunk 3]
```diff
@@ -27,7 +28,11 @@ def get_git_repo_dir():
...
+    encoding = encoding if encoding else 'UTF-8'
+
+    return repo_dir.decode(encoding) if repo_dir and encoding else repo_dir
```
Comment (sergray) [inline pkgversion/pkgversion.py]: It is better ensure, that `repo_dir` is not returned as byte string in Python 3, even if there's no encoding

**Pull request merged**